src/CommandingOfficers/Commander.java

```diff
@@ -43,12 +62,15 @@ public Commander(CommanderInfo info)
     modifiers = new ArrayList<COModifier>();
     units = new ArrayList<Unit>();
     money = DEFAULTSTARTINGMONEY;
+    starsMinor = -1;
```

Comment:
It probably should be in COinfo, but I didn't think about that since I didn't make it and I was focusing on other stuff.